.github/workflows/ci_cpu.yml

```diff
@@ -36,8 +36,12 @@ jobs:
         run: |
           python ts_scripts/install_dependencies.py --environment=dev
       - name: Torchserve Sanity
-        run: |
-          python torchserve_sanity.py
+        uses: nick-fields/retry@v2
+        with:
+          timeout_minutes: 60
+          max_attempts: 3
```

Comment:
Strange how the Mac and Ubuntu time are now taking over an hour, so there's a retry happening but no failures?

Reply:
Here it seems like its 60 minutes for each run and then it timeouts and tries again. I'm not sure why each run is taking more than 60 minutes. As per previous runs, its around 25 minutes